django/db/models/fields/__init__.py

```diff
@@ -711,7 +711,9 @@ def get_db_prep_lookup(self, lookup_type, value, connection,
             return QueryWrapper(('(%s)' % sql), params)
```

Comment:
For clarity, I think it'd be nice to commit the changes here and in `django/db/models/lookups.py` separately as it seems like a refactoring that's independent (maybe a prerequisite) of the changes in contrib.postgres.